lib/cli/ui/widgets/status.rb

```diff
@@ -0,0 +1,53 @@
+# frozen-string-literal: true
+require('cli/ui')
+
+module CLI
+  module UI
+    module Widgets
+      class Status < Widgets::Base
+        ARGPARSE_PATTERN = %r{
+          \A (?<succeeded> \d+)
+          :  (?<failed>    \d+)
+          :  (?<working>   \d+)
+          :  (?<pending>   \d+) \z
+        }x # e.g. "1:23:3:404"
+
+        def render
+          bold_reset = Color::RESET.code + Color::BOLD.code
+
+          +''  \
+            << bold_reset     << '[' << Color::RESET.code \
+            << succeeded_part << ' ' \
```

Comment:
I like what you did, but I wanted to try as well 😄 
How about:
```rb
OPEN = Color::RESET.code + Color::BOLD.code + '[' + Color::RESET.code
CLOSE = Color::RESET.code + Color::BOLD.code + ']' + Color::RESET.code

def render
  parts = [succeeded_part, failed_part, working_part, pending_part]
  OPEN + parts.join(' ') + CLOSE
end
```

Reply:
Hm, yeah, that might be more clear